components/src/dynamo/planner/plugins/transport/_grpc_base.py

```
@@ -0,0 +1,221 @@
+
+    async def close(self) -> None:
+        # Idempotent
+        if self._closed:
+            return
+        self._closed = True
+        if self._channel is not None:
+            try:
+                await self._channel.close()
+            except Exception:
+                # close should never raise to caller
+                pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)